internal/status/status.go

```diff
@@ -44,9 +45,9 @@ const (
 )
 
 type ServerItem struct {
```

Comment:
does ServerItem need to be exported?

Reply:
I thought it does for json marshaller to work properly but seems like that's not true.

Reply:
The fields of the struct have to be exported, but the struct itself does not.